reference.md

```diff
@@ -111,6 +111,10 @@ function
 function call
 :   A use of a function in another piece of software.
 
+heat map
+:   A graphical representation of data in two-dimensions,
+    where data values within the matrix are transformed into a color scale.
```

Comment:
I'm not sure if everyone knows what a color scale is. I'd say that heat map is a graphical representation of two-dimensional data, where different data values are represented with different colors. Perhaps, @ldko has a better suggestion.

Reply:
I don't think "represented with different colors" is as accurate as "color scale", as it doesn't indicate/imply the variation in colors providing meaning to the visualization. If wanting to avoid "color scale", another suggestion:  "A graphical representation of two-dimensional data in which colors, ranging on a scale of hue or intensity, represent the data values."

Reply:
Thanks, Lauren. I like your suggestion -- it's a bit more involved but is more accurate, which is probably what we want for the glossary.

Reply:
That sounds good to me. I'll make the change now.